src/main/scala/it/pagopa/interop/authorizationprocess/common/Adapters.scala

```diff
@@ -20,13 +22,33 @@ object Adapters {
     )
   }
 
+  implicit class ManagementClientWrapper(private val p: AuthorizationManagementDependency.Client) extends AnyVal {
+    def toApi(showRelationShips: Boolean): Client = Client(
+      id = p.id,
+      name = p.name,
+      description = p.description,
+      consumerId = p.consumerId,
+      purposes = p.purposes.map(_.states.toApi),
```

Comment:
I may miss something here, but shouldn't this be 
```suggestion
      purposes = p.purposes.map(_.toApi),
```
?